app/src/store/wallet.ts

```diff
@@ -166,6 +172,7 @@ export default function useWalletStore() {
 
     // Get ENS name if user is connected to mainnet
     const chainId = _provider.network.chainId; // must be done after the .getNetwork() call
+    console.log('connected on chainId: ', chainId);
```

Comment:
This is temporary just to help you verify the network you're connected to with walletconnect, and should be removed before merging